src/ListviewSort.webmodeler.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, createElement } from "react";
+import { findDOMNode } from "react-dom";
+
+import { Dropdown } from "./components/Dropdown";
+import { ValidateConfigs } from "./components/ValidateConfigs";
+import { CommonProps, ListviewSortProps, ListviewSortState, parseStyle } from "./utils/ContainerUtils";
+
+declare function require(name: string): string;
+
+// tslint:disable-next-line class-name
+export class preview extends Component<ListviewSortProps, ListviewSortState> {
+    constructor(props: ListviewSortProps) {
+        super(props);
+
+        this.state = { findingWidget: true };
+    }
+
+    render() {
+        return createElement("div", { className: "widget-offline-search" },
+            createElement(ValidateConfigs, {
+                ...this.props as ListviewSortProps,
+                inWebModeler: true,
+                queryNode: this.state.targetNode,
+                targetGrid: this.state.targetGrid,
```

Comment:
Rename everywhere in this widget something with `Grid`. Either more generic like `targetWidget` or more specific `targetListView`

Reply:
done